ctk/common/src/main/kotlin/org/codice/compliance/verification/core/internal/StatementVerifier.kt

```diff
@@ -84,8 +86,10 @@ internal class StatementVerifier(val node: Node) {
 
             if (encryptedData
```

Comment:
❗️ Since I am removing this check in my PR it may be better to either take this out completely or leave it alone.

Reply:
I'm technically not changing any functionality here. I guess the formatter must've kicked in. I could take this out.